src/components/Navigation.tsx

```diff
@@ -0,0 +1,47 @@
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  return (
+    <nav>
+      <NavLink
+        to={`video-check`}
+        className={({ isActive }) => (isActive ? 'active' : 'nav')}
```

Comment:
Nit: You can probably extract this function outside of the component and then pass a reference to each NavLink.